kuka_kr270_support/urdf/kr270r2700_macro.xacro

```diff
@@ -0,0 +1,182 @@
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/link_5.dae"/>
+        </geometry>
+        <xacro:material_kuka_orange />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/collision/link_5.stl"/>
+        </geometry>
+      </collision>
+    </link>
+    <link name="${prefix}link_6">
+      <visual>
+        <origin xyz="0 0 0 " rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/visual/link_6.dae"/>
+        </geometry>
+        <xacro:material_kuka_black />
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr270_support/meshes/kr270r2700/collision/link_6.stl"/>
+        </geometry>
+      </collision>
+    </link>
+
+    <!-- joints -->
+    <joint name="${prefix}joint_a1" type="revolute">
+      <origin xyz="0 0 0.675" rpy="0 0 0"/>
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}link_1"/>
+      <axis xyz="0 0 -1"/>
+      <limit lower="${radians(-185)}" upper="${radians(185)}" effort="0" velocity="${radians(105)}"/>
+    </joint>
+    <joint name="${prefix}joint_a2" type="revolute">
+      <origin xyz="0.350 0 0" rpy="0 0 0"/>
+      <parent link="${prefix}link_1"/>
+      <child link="${prefix}link_2"/>
+      <axis xyz="0 1 0"/>
+      <limit lower="${radians(-140)}" upper="${radians(-5)}" effort="0" velocity="${radians(101)}"/>
+    </joint>
+    <joint name="${prefix}joint_a3" type="revolute">
+      <origin xyz="1.150 0 0" rpy="0 0 0"/>
+      <parent link="${prefix}link_2"/>
+      <child link="${prefix}link_3"/>
+      <axis xyz="0 1 0"/>
+      <limit lower="${radians(-120)}" upper="${radians(150)}" effort="0" velocity="${radians(107)}"/>
+    </joint>
+    <joint name="${prefix}joint_a4" type="revolute">
+      <origin xyz="1.200 0 -0.041" rpy="0 0 0"/>
+      <parent link="${prefix}link_3"/>
+      <child link="${prefix}link_4"/>
+      <axis xyz="-1 0 0"/>
+      <limit lower="${radians(-350)}" upper="${radians(350)}" effort="0" velocity="${radians(122)}"/>
+    </joint>
+    <joint name="${prefix}joint_a5" type="revolute">
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <parent link="${prefix}link_4"/>
+      <child link="${prefix}link_5"/>
+      <axis xyz="0 1 0"/>
+      <limit lower="${radians(-122.5)}" upper="${radians(122.5)}" effort="0" velocity="${radians(113)}"/>
+    </joint>
+    <joint name="${prefix}joint_a6" type="revolute">
+      <origin xyz="0 0 0" rpy="0 0 0"/>
+      <parent link="${prefix}link_5"/>
+      <child link="${prefix}link_6"/>
+      <axis xyz="-1 0 0"/>
+      <limit lower="${radians(-350)}" upper="${radians(350)}" effort="0" velocity="${radians(175)}"/>
+    </joint>
+
+    <!-- This frame corresponds to the $TOOL coordinate system in KUKA KRC controllers. -->
```

Comment:
Both `flange` and `tool0` have this comment.

Is that correct?

Reply:
Seems to be the case for the other robots as well, see e.g. [KR150-2](https://github.com/ros-industrial/kuka_experimental/blob/melodic-devel/kuka_kr150_support/urdf/kr150_2_macro.xacro)

However, the KR150 r3100 has the following comment on `flange`:

    <!-- ROS-Industrial 'flange' frame: attachment point for EEF models -->

and `tool0`

    <!-- ROS-Industrial 'tool0' frame: all-zeros tool frame -->
    <!-- This frame corresponds to the $FLANGE coordinate system in KUKA KRC controllers. -->

Unfortunately, I have no idea what is correct / how to check that

Reply:
Sorry for letting this get stale again, and sorry for getting sloppy with the comment for KR150-2 in #195 

You could use either the comments you mentioned here or like in the PR above (REP-199 ...)